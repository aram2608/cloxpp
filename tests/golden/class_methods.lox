class Greeter {
  sayHi() {
    print "Hi!";
  }
}
Greeter().sayHi(); // expect "Hi!"