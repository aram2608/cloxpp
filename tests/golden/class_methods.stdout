Token: CLASS class 1
Token: LEFT_BRACE { 1
Token: IDENTIFIER sayHi 2
Token: LEFT_PAREN ( 2
Token: RIGHT_PAREN ) 2
Token: LEFT_BRACE { 2
Token: PRINT print 3
Token: STRING "Hi!" 3
Token: SEMICOLON ; 3
Token: RIGHT_BRACE } 4
Token: RIGHT_BRACE } 5
Token: LEFT_PAREN ( 6
Token: RIGHT_PAREN ) 6
Token: DOT . 6
Token: IDENTIFIER sayHi 6
Token: LEFT_PAREN ( 6
Token: RIGHT_PAREN ) 6
Token: SEMICOLON ; 6
Token: EOF  6
